src/planet-4-151612/wordpress/etc/my_init.d/90_wordpress.sh

```diff
@@ -3,11 +3,14 @@ set -euo pipefail
 
 # Executed after my_init.d and environment is established
 
-# Generates keys and salts for wp-config.php
-/app/bin/generate_wp_keys.sh
+# Create wp-config.php file, overwrite if not in development environment
+if [[ ! -f "${PUBLIC_PATH}/wp-config.php" ]] || [[ "${APP_ENV}" != "develop"* ]]; then
```

Comment:
I assume this will also affect dev sites 🤔

Reply:
@comzeradd  If their files aren't reinstalled from scratch on each deployment yes - I was looking into adding a "local" keyword for APP_ENV, but it would be a bit more work, I can do that if you prefer

Reply:
We do build from scratch each time, so this should be fine even for dev sites 🤞🏻